src/InputFilter.cpp

```diff
@@ -108,7 +108,7 @@ namespace
  * This delay in events is ordinarily not noticable, because we report initial
  * presses and releases immediately.  However, if a real press is ever delayed,
  * this won't cause timing problems, because the event timestamp is preserved. */
-static Preference<float> g_fInputDebounceTime( "InputDebounceTime", 0 );
+static Preference<float> g_fInputDebounceTime( "InputDebounceTime", 0.02 );
```

Comment:
I would update the comment why we're setting it to 20ms.

Essentially "For pad play, a value of 20ms-50ms seems to result in a better experience. For keyboard play, this is often set to 0." or similar

Reply:
Do you mean the code comment or the description in SL?

Reply:
Code comment in InputFilter.cpp

Reply:
updated!